integration-tests/src/test/scala/za/co/absa/spline/harvester/LineageHarvesterSpec.scala

```diff
@@ -52,7 +53,7 @@ class LineageHarvesterSpec extends AsyncFlatSpec
       withLineageTracking { captor =>
         import spark.implicits._
 
-        val testDest = tmpDest
+        val testDest = tmpDest.asString
```

Comment:
In this case the code would be clearer using `tmpDest()` to make it known that method is called. I thought it was variable for some time.

Also it is recomended in scala style guide
https://docs.scala-lang.org/style/method-invocation.html#arity-0

Reply:
or you could just delete the method and use the code directly it's really short and it would make it obvious what is happening.

Reply:
Fixed in commit 5d0f9ae49f2b674bea77d05b7807d51df7eacc06.